playkit/src/main/java/com/kaltura/playkit/PlayerEvent.java

```diff
@@ -113,10 +113,12 @@ public MetadataAvailable(List<PKMetadata> metadataList) {
     public static class SourceSelected extends PlayerEvent {
 
         public final String sourceUrl;
+        public final String sourceExtention;
 
-        public SourceSelected(String sourceUrl) {
+        public SourceSelected(String sourceUrl, String sourceExtention) {
```

Comment:
I think we might want to extend this, either add the source object to the event, or maybe expose an API to get the object on the player and when the event is triggered a listener will use the API to get the info.
@noamtamim, @giladna WDYT?